upload/src/addons/SV/TitleEditHistory/_output/content_type_fields/thread_title-edit_history_handler_class.json

```diff
@@ -0,0 +1,5 @@
+{
+    "content_type": "thread_title",
+    "field_name": "edit_history_handler_class",
+    "field_value": "SV\\TitleEditHistory\\EditHistoryHandler\\Thread"
```

Comment:
Rename from EditHistoryHandler => EditHistory, and thankfully it doesn't require a custom upgrade step (unlike XF1)